src/js/vic-v2/config/form.js

```diff
@@ -159,10 +159,12 @@ const formConfig = {
     },
     documentUpload: {
       title: 'Document Upload',
+      reviewTitle: 'Document Review',
```

Comment:
I think adding a reviewTitle is fine, but I wonder if it would be better in this case to just have "Documents" be the chapter title in both places?

Reply:
Good idea, @peggygannon is it preferred to have distinct titles for "Document upload" in the form and "Document review" on the review page, or could we simply use "Documents" as the chapter title in both places?

Reply:
@cehsu I think we should keep the earlier chapter headers as "Document Upload", to keep with the style of other forms. And I think it's fine to change the header to "Documents" on the review title.

Reply:
Thanks @peggygannon, updated.